pkg/resourcemonitor/resourcemonitor_test.go

```diff
@@ -691,23 +691,23 @@ func TestResourcesScan(t *testing.T) {
 					Resources: topologyv1alpha1.ResourceInfoList{
 						topologyv1alpha1.ResourceInfo{
 							Name:        "cpu",
-							Allocatable: intstr.FromString("10"),
-							Capacity:    intstr.FromString("12"),
+							Allocatable: resource.MustParse("10"),
```

Comment:
Wanted to point out how I went ahead with this change in NFD, I assumed that capacity=allocatable (i.e. we don't account for kube-reserved and system reserved) and changed the values in the `allocatable` field to be captured in the `available` field some like I did here: https://github.com/kubernetes-sigs/node-feature-discovery/pull/525/commits/041aa80b31a57dab0832d10081ef19a3ffd1ef9e and then later in a subsequent commit took the reserved resources into consideration: https://github.com/kubernetes-sigs/node-feature-discovery/pull/525/commits/041aa80b31a57dab0832d10081ef19a3ffd1ef9e.